Documentation/dev-guide/limit.md

```diff
@@ -2,7 +2,7 @@
 
 ## Request size limit
 
-etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. For the time being, etcd guarantees to support RPC requests with up to 1MB of data. In the future, the size limit may be loosened or made configurable.
```

Comment:
the sever side data limit is still 1MB i believe.

Reply:
@xiang90 It's 1.5 MiB now? https://github.com/coreos/etcd/pull/7968

2 MiB was set default 1.5 MiB + gRPC overhead.

Reply:
Users will not care about what gRPC limits. They care about the size of key+value in put or in transaction. let us put 1.5Mi there, and keep the previous working (not mentioning gRPC client thing here.)